src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java

```diff
@@ -2130,6 +2130,22 @@ public HyperwalletTransferMethod createTransferMethod(String jsonCacheToken, Str
                 headers);
     }
 
+    /**
+     * List Transfer Methods
+     *
+     * @param userToken String user token
+     * @param options   List filter option
+     * @return HyperwalletList of HyperwalletTransferMethod
+     */
+    public HyperwalletList<HyperwalletTransferMethod> listTransferMethods(String userToken, HyperwalletPaginationOptions options) {
+        String url = paginate(this.url + "/users/" + userToken + "/transfer-methods", options);
```

Comment:
seems like this url concat pattern repeats quite often. we should consider creating a UrlBuilder (using the builder patter)